lib/msf/core/exploit/remote/kerberos/ticket/storage/read_mixin.rb

```diff
@@ -0,0 +1,24 @@
+module Msf::Exploit::Remote::Kerberos::Ticket::Storage
+  module ReadMixin
+    def load_credential(options = {})
+      return nil unless active_db?
+
+      now = Time.now.utc
+      tickets(options) do |ticket|
+        next if ticket.expired?(now)
+
+        return ticket.ccache.credentials.first
+      end
+
+      nil
+    end
+
```

Comment:
```suggestion
# @see Msf::Exploit::Remote::Kerberos::Ticket::Storage#tickets
```

As a compromise for users to see that there's docs at least somewhere 😅 

Maybe there's a better yard doc syntax we can use though